schemas/aws/terraform-state-1.yml

```diff
@@ -0,0 +1,20 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+description: bucket properties for each integration
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /aws/terraform-state-1.yml
+  bucket:
+    type: string
+  key:
+    type: string
+  region:
+    type: string
+  integration:
```

Comment:
what is the `integration` field for? mind adding descriptions?

Reply:
so each /dependencies/terraform-state-1.yml object is a bucket/key reference for a specific integration and a list of those can be referenced from an /aws/account-1.yml, right?

an alternative would have been to reference terraform-state-1 objects as a dictionary from an account, having the key an integration name and the value a terraform-state-1 object. but this would imply updating the account schema every time we have a new integration.